docs/user-guide/metadata.md

```diff
@@ -190,3 +192,16 @@ jobs:
 
 Result:
 ![Label](./assets/label-meta.png)
+
+### Notification
+
+You can use [notification](./configuration/settings.html#slack) function from steps.
```

Comment:
Is this better?
`You can custom message for [notification](./configuration/settings.html#slack). Custom content is different for each notification plugins.`

Reply:
Thanks, I fixed like you told.